src/config.js

```diff
@@ -164,7 +164,7 @@ async function productionBranchesConfig(url) {
 
     let repourl = /^https:\/\/(hg\.mozilla\.org\/.*)$/.exec(pb.repo);
     if (!repourl) {
-      debug('skipping production branch ' + alias + ': unrecognizede repo URL ' + pb.repo);
+      debug('skipping production branch ' + alias + ': unrecognized repo URL ' + pb.repo);
```

Comment:
Not related to review: Should we alert to sentry when there are unrecognized things rather than just debug?

Reply:
No, I think it wouldn't be expected for mozilla-taskcluster to handle such repos.